moveit_ros/moveit_servo/src/collision_check.cpp

```diff
@@ -203,6 +207,16 @@ void CollisionCheck::run(const ros::TimerEvent& timer_event)
   }
 }
 
+void CollisionCheck::printCollisionPairs(collision_detection::CollisionResult::ContactMap& contact_map)
+{
+  if (!collision_result_.contacts.empty())
+  {
+    ROS_ERROR_STREAM_THROTTLE_NAMED(ROS_LOG_THROTTLE_PERIOD, LOGNAME,
+      "Ojects in collision (among others, possibly): " << collision_result_.contacts.begin()->first.first << ", "
+      << collision_result_.contacts.begin()->first.second);
+  }
+}
+
```

Comment:
This is nice, if we get here it'd be nice to know what is in collision in the logs.

Reply:
This is really helpful. Maybe we should print all collisions but make it a DEBUG statement?

Reply:
Just noticed that the argument here isn't being used. Will fix that

Reply:
@henningkayser I tried to think about how I would prefer to go about debugging collisions. You're right, I think printing every collision pair might be too much. But switching to DEBUG logging is a little bit of a pain, and it may be an unexpected collision and the user was not in DEBUG mode beforehand. Also, I'm not sure how many ROS users even know how to switch console logging modes.

Reply:
I would consider using the debug log a ROS basic so for me it's more a matter of usefulness. For debugging collisions DEBUG seems like the perfect logger level. We should still have a (throtttled) warning if we are decelerating and there we could add the number of collisions and possibly the first collision pair.

Reply:
done